packages/manager/apps/dedicated/client/app/dedicated/server/netboot/module.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
```

Comment:
Missing imports related to translations:
`import '@ovh-ux/ng-translate-async-loader';`
`import 'angular-translate';`

Reply:
Its defined in the parent modules.